eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,11 @@
+-SourceToolsList @("policheck","credscan")
+-TsaInstanceURL https://devdiv.visualstudio.com/
+-TsaProjectName DEVDIV
+-TsaNotificationEmail aspnetcore-build@microsoft.com
+-TsaCodebaseAdmin REDMOND\kevinpi
+-TsaBugAreaPath "DevDiv\ASP.NET Core"
+-TsaIterationPath DevDiv
+-TsaRepositoryName AspNetCore
+-TsaCodebaseName AspNetCore
+-TsaOnboard $True
```

Comment:
What does this new setting do?

Reply:
I don't know what it does, but it was present in the old variable group. I guess we just didn't pass it - @jcagme?

Reply:
@sunandabalu @jonfortescue  should have more info in terms of what each arguments does

Reply:
TsaOnboard is the variable that tells the scripts to onboard the account in TSA if not already done, this is optional as by default its set to true in the scripts. TSA is the automation that takes care of filing issues in the the instanceurl specified.